### PR TITLE
Fixes the styling of the job cards

### PR DESCRIPTION
**Pull request by substrae**
image

## ✨ Summary

[Visit the Figma file](https://www.figma.com/file/guHeIIh8dqFVCks310Wv0G/Style-library?type=design&node-id=5-12&mode=design) • Contact @substrae with design questions

This pull request addresses the styling issues with job cards on the "Browse jobs" page.

```[tasklist]
### ✅ Review requirements
- [ ] Purge cash and rerun maintenance scripts
- [ ] Run dev
- [ ] Check out the browse jobs page on desktop and mobile
- [ ] run Storybook
- [ ] Check out the pool card component on desktop and mobile
``` 

## 🔍 Details

<details>
  <summary>📸 Before and after</summary>
  
</details>

### CODE REVIEW

File: apps/web/src/components/PoolCard/PoolCard.tsx [hunk 1]
```diff
@@ -62,84 +62,106 @@ const PoolCard = ({ pool, headingLevel = "h3" }: PoolCardProps) => {
...
-          //   clipPath: `polygon(0% 0%, 0% 100%, 50% calc(100% - 2rem), 100% 100%, 100% 0)`,
-          // }}
+        <span
+          data-h2-color="base(black) base:iap(white)"
+          data-h2-font-weight="base(700)"
+          data-h2-font-size="base(h5, 1)"
+          data-h2-layer="base(2)"
+          data-h2-position="base(absolute)"
+          data-h2-location="base(auto, auto, x1.25, 50%)"
+          data-h2-transform="base(translate(-50%, 0px))"
```
Comment (esizer) [inline apps/web/src/components/PoolCard/PoolCard.tsx:94]: Something here is causing the flag text to wrap for me 😢 

Screenshot 2023-10-06 103641

**Review by esizer — CHANGES_REQUESTED** [review]
Looks great! Just one minor wrapping issue with the flag.

File: apps/web/src/components/PoolCard/PoolCard.tsx (hunk 1 above)
Comment (substrae) [reply apps/web/src/components/PoolCard/PoolCard.tsx:92]: @esizer Should be fixed now :D

**Review by esizer — APPROVED** [review]
LGTM! 👍

**Pull request merged**